recipes/mecab-ko/run_test.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh 
+
+set -e
+set +x
+
+$PREFIX/bin/mecab -v
+$PREFIX/bin/mecab -h || true
```

Comment:
To ensure this line actually tests something can you do a test like?
```suggestion
$PREFIX/bin/mecab -h | grep something
```
(I assume it returns a code of 1 instead of 0)